emcc.py

```diff
@@ -1318,17 +1319,111 @@ def get_bitcode_args(input_files):
         logging.debug(('just preprocessor ' if '-E' in newargs else 'just dependencies: ') + ' '.join(cmd))
         exit(subprocess.call(cmd))
 
-      def compile_source_file(i, input_file):
+      def compile_source_file(i, input_file, browsix_file = None):
         logging.debug('compiling source file: ' + input_file)
         output_file = get_bitcode_file(input_file)
-        temp_files.append((i, output_file))
+        if (browsix_file is None):
+          temp_files.append((i, output_file))
         args = get_bitcode_args([input_file]) + ['-emit-llvm', '-c', '-o', output_file]
         logging.debug("running: " + ' '.join(shared.Building.doublequote_spaces(args))) # NOTE: Printing this line here in this specific format is important, it is parsed to implement the "emcc --cflags" command
         execute(args) # let compiler frontend print directly, so colors are saved (PIPE kills that)
         if not os.path.exists(output_file):
           logging.error('compiler frontend failed to generate LLVM bitcode, halting')
           sys.exit(1)
-
+      
+      def get_llvm_ctors_dtors (i, input_file):
```

Comment:
nit: remove space between `dtors` and args:

    def get_llvm_ctors_dtors(i, input_file):